### DISCUSSION TITLE
Reclamación en un video de PS2

### DISCUSSION THREAD

**Discussion by TheWizWikii** [israpps/Funtuna-Fork#13] [General]
Le escribo ya que he recibido una reclamación por su parte en un video mio de youtube titulado FMCB en PS2 SLIM 9XXXX con FunTuna FORK. 

¿Hay algún motivo por el cual ha reclamado el video? Puse información no válida o incorrecta? ¿Es usted el desarrollador del proyecto?
Si lo es, hay alguna forma en que podamos quedar de acuerdo para que retire la reclamación. Estaría dispuesto a modificar el video si fuera necesario.

Espero su respuesta

Atentamente

TheWizWiki

**Comment by TheWizWikii** [comment]
No vi esta parte:

**redistribución**:
se le permite redistribuir este paquete y todo su contenido SOLO si utiliza el [servicio de descarga principal](https://github.com/israpps/Funtuna-Fork/releases) ubicado en la sección Lanzamiento de este repositorio, y NO desde ningún otro servicio de descarga (como: mega.nz mediafire.com y cualquier servicio de naturaleza similar)


 o cuando yo subí el video no estaba puesto.

No se porque reclamas el video, podrías haberme contactado directamente y lo cambiaba de inmediato poniendo tus créditos y enlace directo al repositorio que es lo que suelo hacer.

**Comment by israpps** [comment]
@TheWizWikii hola.

Te lo mencioné en tu vídeo dos días después de que lo subieras.

Me ignoraste totalmente en el vídeo, alegando que _ahora que existía Opentuna funtuna ya no sirve_.

De hecho, está reclamación fue creada una semana después de que publicarlas ese vídeo, se tardó por la ineficiencia de Google debido a la pandemia.

Ignoraron dos correos míos, hasta la semana pasada (la 3ra es la vencida)

Tu vídeo ya está prácticamente desactualizado.

FunTuna cambió bastante en estos meses, contando ahora con un instalador automático, soporte para todas las ps2 excepto las ProtoKernel (`SCPH-1XXXX`)
Así como varios ajustes de estabilidad.

De hecho, se acerca la versión 2.0.0 (#4 ) con un Port del FreeMcBoot y su configurador al último SDK

**Comment by TheWizWikii** [comment]
Gracias por tu rápida respuesta.

No es que ignorara tu mensaje o comentario, posiblemente no me fije o no lo
leí.

Hay alguna esperanza de que podamos llegar a un acuerdo? El vídeo ya era
algo antiguo y me imagino que ahora habrán cambiado las cosas. Serías tan
amable de retirar la reclamación? Estaría encantado de volver a cacharrear
con tu repositorio y actualizar vídeo edición 2022

Atentamente

TheWizWhisky

**Reply by israpps** [reply]
Puedo sacarlo...

Si eliminas los enlaces alternos de tu vídeo...

Y de esa página, it's time 2 play

**Comment by israpps** [comment]
Si te interesa también podés hacer videos de alguno de mis otros proyectos, que no tienen la difusión que merecen.

Cómo HDL batch installer, mis instaladores personalizados de FreeMcBoot, o mis mod de uLaunchELF.

**Comment by TheWizWikii** [comment]
Por supuesto que lo retiró.

Lo que haré automáticamente enlazar el repositorio de GitHub. Además de eso
intentaré hacer un video actualizado de fortuna 2022 con esto Releases ·
israpps/Funtuna-Fork (github.com)
<https://github.com/israpps/Funtuna-Fork/releases> si me acuerdo cómo
funcionaba xD

Atentamente

TheWizWhisky

**Comment by TheWizWikii** [comment]
Sobre esto: Si te interesa también podés hacer videos de alguno de mis
otros proyectos, que no tienen la difusión que merecen.

Cómo HDL batch installer, mis instaladores personalizados de FreeMcBoot, o
mis mod de uLaunchELF.


Podría mirar de traer algo ya que parece interesante


Atentamente


TheWizWhisky
